planet/order_request.py

```diff
@@ -199,13 +199,17 @@ def delivery(archive_type: Optional[str] = None,
     if archive_type:
         archive_type = specs.validate_archive_type(archive_type)
 
-    fields = ['archive_type', 'single_archive', 'archive_filename']
-    values = [archive_type, single_archive, archive_filename]
+    config: Dict[str, Any] = {}
 
-    config = dict((k, v) for k, v in zip(fields, values) if v)
+    if archive_type:
+        config.update(archive_type=archive_type,
+                      archive_filename=archive_filename
+                      or '{{name}}-{{order_id}}.zip',
```

Comment:
Here's a default archive filename.

Reply:
Note that unless the archive type is given, there are no archive items in the "delivery" config.